### DISCUSSION TITLE
Github Action: Support for workflow level env for reusable workflows

### DISCUSSION THREAD

**Discussion by elben10** [community/community#6110] [Actions]
Currently it is not possible to pass values from env context (workflow level) to reusable workflows. By enabling support for workflow env context together with resuable workflows would result in one being able to define variables once that is used by multiple reusable workflows.

The following repository and belonging actions reproduces the issue

[Action Run](https://github.com/elben10/action-composition/actions/runs/1319321083/workflow)
[Repository](https://github.com/elben10/action-composition)

[Docs - Reusable Workflows](https://docs.github.com/en/actions/learn-github-actions/reusing-workflows)

**Comment by glloydcorecard** [comment]
This is absolute must have. Right now I am needing to define a tag version 7 times. This should simply defined once as a workflow env variable then provided to the reusable workflow via the `with` section. Being limited to only the `${{ github.* }}` set of functions seems like a major oversight.

**Comment by OneCyrus** [comment]
not being able to use env variables as input to reusable workflows is very limiting. this blocks us to create generic reusable workflows.
